app/components/menu.tsx

```diff
@@ -0,0 +1,105 @@
+"use client";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+
+import Typography from "@mui/material/Typography";
+
+import Button from "@mui/material/Button";
+
+import React from "react";
+import Image from "next/image";
+import clinic from "@/app/assets/clinic.jpg";
+import { Stack } from "@mui/material";
+import { dosis } from "@/app/fonts/font";
+import ButtonTopBar from "@/app/ui/buttonTopBar";
+
+const pages = [
```

Comment:
crie um arquivo em @/constants/Navigation.ts e crie essa array lá

export links = [contato, reviews, galeria, etc]

E depois importe para cá.

Reply:
I put it as .tsx because I need to study how to make a list with pure Typscript.